JustSaying/Extensions/TimespanExtensions.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Globalization;
+
+namespace JustSaying.Extensions
+{
+    public static class TimespanExtensions

```

Comment:
If you add an attribute [like this](https://github.com/martincostello/sqllocaldb/blob/d734da85cd2e4355a3e662df7d4b73b9c4e5dd55/src/SqlLocalDb/ISqlLocalDbInstanceInfoExtensions.cs#L13) you can hide the class from Intellisense so people don't get suggestions to write code like `TimespanExtensions.AsSecondsString(TimeSpan.Zero)`.